crates/fj/Cargo.toml

```diff
@@ -16,7 +16,10 @@ default = []
 serialization = ["serde"]
 
 [dependencies]
-serde = { version = "1.0.7", optional = true }
+serde = { version = "1.0.7", features = ["derive"], optional = true }
+
+[dev-dependencies]
+serde_json = "1.0.81"
```

Comment:
Minor nitpick: I think it would be a bit more clear to move this to the end of the file. Otherwise we have `dependencies` inter-mixed with `dev-dependencies`.